crates/rust-analyzer/src/bin/args.rs

```diff
@@ -74,10 +74,11 @@ impl Args {
             None => {
                 if matches.contains(["-h", "--help"]) {
                     print_subcommands();
-                    return Ok(Args { verbosity, command: Command::Help });
+                    String::from("help")
+                } else {
+                    matches.finish().or_else(handle_extra_flags)?;
+                    return Ok(Args { verbosity, command: Command::RunServer });
                 }
-                matches.finish().or_else(handle_extra_flags)?;
-                return Ok(Args { verbosity, command: Command::RunServer });
             }
```

Comment:
The early return versions reads better to me, and it also seems less verbose. Duplicating `verbosity` in unfortuante, but berpapse we can just `let help = ...; return help`?

Reply:
Done

Reply:
bors r+